### PR TITLE
Dropped uniqueness constraint on user email

### LINKED ISSUE #2387: Drop Bodhi's uniqueness constraint on e-mails
Bodhi currently has a uniqueness constraint on e-mails. This causes problems when users have more than one account and then change an account's e-mail address to be the same as the other accounts. When they log into Bodhi, they will get a crash because Bodhi hits the e-mail field's uniqueness constraint check. Bodhi does not look up users by e-mail address, and only uses the field to send e-mails, so it's ok if two accounts have the same address. We don't even need to index the field because we don't look up or search users by e-mail address.

### PR DESCRIPTION
**Pull request by ghost**
fixes #2387

Signed-off-by: Vismay Golwala <vgolwala@redhat.com>

### CODE REVIEW

File: bodhi/server/migrations/versions/39132c56a47e_drop_uniqueness_constraint_on_user_email.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2017 Red Hat, Inc.
```
Comment (bowlofeggs) [inline bodhi/server/migrations/versions/39132c56a47e_drop_uniqueness_constraint_on_user_email.py]: Let's update the year to 2018.

**Review by bowlofeggs — CHANGES_REQUESTED** [review]

**Comment by vismay-golwala** [comment]
jenkies test

**Comment by vismay-golwala** [comment]
Jenkies test

**Comment by bowlofeggs** [comment]
jenkies test

**Review by bowlofeggs — APPROVED** [review]
This LGTM, but should not be merged until #2454 is merged because it references a ```down_revision``` that is created there.

**Pull request merged**